cluster/hooks.go

```diff
@@ -443,6 +445,21 @@ func InstallKubernetesDashboardPostHook(input interface{}) error {
 			return err
 		}
 
+		// set admin rights (if needed)
```

Comment:
We should probably move this to its own hook and call it right after `StoreKubeConfig`. The admin rights could also be necessary elsewhere.